python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -78,46 +81,54 @@ def _isOdd(x: int) -> bool:  # noqa: N805
     sigmas = pexConfig.ListField(
         dtype=float,
         default=[4.0, 5.0],
-        doc="List of sigmas (in pixels) of circular Gaussian apertures to apply on "
+        doc=("List of sigmas (in pixels) of circular Gaussian apertures to apply on "
```

Comment:
Multi-line docs for config fields is rare, and I have seen examples with and without parenthesis in the codebase. I couldn't find an explicit instruction in the dev-guide and I felt safer putting them within parenthesis. Happy to remove it if that's preferred.

Reply:
I also don't see what extra whitespace you're referring to. Can you suggest the change you're referring to directly on the PR?

Reply:
As far as I can tell, since this is already inside parentheses the opening bracket is not needed for that string. A quick test of:

```python
def a(b):
    print(b)

a(b="a long string"
    " that continues")
```
does work fine without needing the extra `()`

Reply:
Right, I was just uneasy seeing strings concatenated just like that. I did have the multi-line docs without extra parenthesis but given the general rule in the dev-guide that we should use minimal parenthesis required, I'll go back to not having those extra parenthesis.